integration/features/chaincode-ops-on-docker.feature

```diff
@@ -47,17 +47,22 @@ Feature: Chaincode ops on docker-based Fabric network
     And chaincode (name: private, seq: 1, channel: mychannel) should be committed over the fabric network
     And chaincode (name: private, channel: mychannel) should be set the collections for private template
 
+    # Set malicious votings to 0
+    When consortium sets max malicous votings (number: 0)
```

Comment:
```suggestion
    When consortium sets max malicious votings (number: 0)
```
This typo seems to appear multiple times (including the step script). Also please refer to my comment regarding "votings"